less/index.less

```diff
@@ -65,112 +65,112 @@ h1, h2, h3, h4, h5 {
 }
 
 .container {
-    width: 880px;
+    width: 800px + 80;
     margin: auto;
 }
 
 /* Header Styles */
 
 header {
-    width: 100%;
+    width: 50% * 2;
```

Comment:
So obviously just multiplying hard-coded values here is a little silly, but it's a good way to play with LESS and prove to yourself its functionality. So, where would you want to use these math operators? Well, let's say you had a parameter that you were using to designate widths. You could set everything on your page as a multiple of that width, so for example, you could easily set two button sizes, one that was 50px and another that was 100px. You could have `@btn-width-baseline = 50px` and then have the other button be the same as a multiple of two. Then, later on, if you ever need to tweak your button size, you could change it to 60px and know that the other button is going to automatically become 120px, keeping the same size ratio. You could do this with colors, too, for example, to always set one color as a lighter shade of another color, for example. Some food for thought.